src/sentry/runner/commands/devservices.py

```diff
@@ -303,6 +303,19 @@ def up(
     """
     from sentry.runner import configure
 
+    click.secho(
+        """
+WARNING: We're transitioning from `sentry devservices` to the new and improved `devservices` in the near future.
+To give the new devservices a try, set the `USE_NEW_DEVSERVICES` environment variable to `1`.
+
+Instead of running `sentry devservices up`, consider using `devservices up`.
+If you hit any bumps or have feedback, we'd love to hear from you — just drop by #discuss-dev-infra.
```

Comment:
Should we make a note like "For employees testing it out:" since non-employees can't reach out on slack

Reply:
Good call